cirq/ops/parity_gates.py

```diff
@@ -110,7 +110,7 @@ def _eigen_components(self):
         ]
 
     def _eigen_shifts(self):
-        return 0, 1
+        return [0, 1]
```

Comment:
Why the square brackets?

Reply:
The return type is List[int]

Reply:
So otherwise the consistent eigenshifts tests would fail

Reply:
Oh, I see.

The return type should be `Sequence[int]`, or maybe even `Iterable[int]`, not `List[int]`. The receiving code shouldn't be depending on the fact that it's a list or a tuple.